test/hotspot/jtreg/serviceability/jdwp/DebuggeeLauncher.java

```diff
@@ -107,12 +101,7 @@ public int getJdwpPort() {
 
     @Override
     public void onStringRead(StreamHandler handler, String line) {
```

Comment:
`handler` seems unused now as well. There was obviously some kind of grander design for this API that we now seem to be cutting off.

Reply:
Submitted https://github.com/openjdk/jdk/pull/15502